packages/main/test/specs/Input.spec.js

```diff
@@ -856,10 +856,8 @@ describe("Input general interaction", () => {
 		//act
 		await inputWithGroupsInnerInput.keys("ArrowDown");
 
-		const announcementText = await inputWithGroupsAnnouncement.getText();
-
 		//assert
-		assert.ok(announcementText.includes("List item 1 of 12"), "The total count announcement and position of items should exclude group items.");
+		assert.strictEqual(await inputWithGroupsAnnouncement.getText(), "travel the world explore List item 1 of 12", "The total count announcement and position of items should exclude group items.");
```

Comment:
The assert and the assert message expect different things - make a new assert specific for testing additional text and description and leave this one as the focus here is on excluding the groups from the item count.

Reply:
Done